package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
Agreed. The readme is a much better place. I am just unsure if we should rely on the people to give up their habits.